readme-developer.md

```diff
@@ -0,0 +1,158 @@
+
+For example, when making lots of changes to the game, you will generally have the "endless-sky-lib" project active, so that you can ensure changes compile without needing to fully link things together into the actual game binary. After making changes, you would then activate the "EndlessSky" project and use the "Build & Run" option with the "Release" target, which will fully link the compiled code into the small, performant executable.
+
+You can continue making tweaks to files that belong to the "endless-sky-lib" project without reactivating it, so that "Build & Run" will still launch the game with your latest tweaks. Should you experience some show-stopper bug in your modified game, you would then change to the "Debug" build target to help determine the source(s) of the issue.
+
+Before the game will run the first time, however, you will need to skip down to the instructions at **Running the game** below.
+
+### Building with Scons
+
+If you instead want to build the game from the command line via Scons, you will need [Python 3.8 or later](https://www.python.org/downloads/). When installing, be sure to select the "Add to PATH" checkbox.
+
+Afterwards, you will need to add your MinGW installation to your path manually. To do so on Windows 10, to to **Settings** > **System** > **About** > **System info** > **Advanced system settings** > **Environment Variables**. From each, select **Path** under System variables, and click Edit.
+
+Click New, and then if you used a default MinGW installation, enter `C:\Program Files\mingw-w64\x86_64-8.1.0-posix-seh-rt_v6-rev0\mingw64\bin` as the path, then hit OK to exit out of these windows.
+
+From there, restart your command line if you had one open, and simply `cd` to your Endless Sky directory, then type `mingw32-make.exe -f .winmake`. To speed up the build process, you can add a `-jX` flag to that command, where X is the number of CORES/THREADS? your processor has.
+
+### Building with a third-party IDE
+Refer to the build pipeline files under `.github\` to tailor the build process to your chosen IDE.
+
+### Running the Game
+
+For Endless Sky to run on Windows, it needs various .dll files included in the base directory. Go to `C:\dev64\bin\`, and copy the files to your base directory. These should include:
+
+  - glew32.dll
+  - **libgcc_s_seh-1.dll**
+  - libjpeg-62.dll
+  - libmad-0.dll
+  - libpng15-15.dll
+  - **libstdc++-6.dll**
+  - libturbojpeg.dll
+  - libwinpthread-1.dll
+  - OpenAL32.dll
+  - SDL2.dll
+  - soft_oal.dll
+  - zlib1.dll
+
+For the **bolded** libraries, those will need to be included from your MinGW installation, instead. By default, you can go to `C:\Program Files\mingw-w64\x86_64-8.1.0-posix-seh-rt_v6-rev0\mingw64\bin`, and copy those libraries to your Endless Sky directory to overwrite the current ones.
+
+**TODO:** Incorporate these libraries into the provided zip, so the user only has to copy from one source.
```

Comment:
This won't be done

Reply:
The only reason mentioned on Discord was we couldn't be sure what version of MinGW/g++ the user was using. Now, we specify for new developers to use 8.1.0. And even in that case, I went back and test 8.1.0, 7.1.0, 6.3.0, and 5.1.0. All compiled without a hitch on both my current installation and on a VM, and the game ran with only needing to copy from one directory. Is there some other reason I'm missing?